### PR TITLE
EXUI-1301 - error message updated

### PR DESCRIPTION
**Pull request by olusegz07**
### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by RiteshHMCTS — DISMISSED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by connorpgpmcelroy — DISMISSED** [review]
Build failing

File: api/registerOrganisation/index.ts [hunk 1]
```diff
@@ -108,3 +110,17 @@ function convertEmptyStringToNull(term: string): string {
   return term === '' ? null : term;
 }
 
+function securePbaNumberErrorMessage(data, errorMesage: string): string {
```
Comment (Josh-HMCTS) [inline api/registerOrganisation/index.ts:113]: Minor typo on errorMesage

**Review by Josh-HMCTS — DISMISSED** [review]

**Review by OgunyemiO — APPROVED** [review]

File: api/registerOrganisation/index.ts [hunk 2]
```diff
@@ -108,3 +110,17 @@ function convertEmptyStringToNull(term: string): string {
...
+  const emailDuplicateErrorMessage = '6 : EMAIL Invalid or already exists';
+
+  if (errorMessage.toLowerCase() === pbaNumberErrorMessage.toLowerCase()) {
+    return 'Registration cannot be completed';
+  }
+
+  if (errorMessage.toLowerCase() === emailDuplicateErrorMessage.toLowerCase()) {
+    return 'A user with this email address already exists. You should check that the email address has been entered correctly. You should also check if your organisation has already been registered. If you\'re still having problems, contact HMCTS.';
+  }
+
```
Comment (andywilkinshmcts) [inline api/registerOrganisation/index.ts:124]: Can you check in here if errorDescription is null , and if so return a default message

**Review by andywilkinshmcts — CHANGES_REQUESTED** [review]

File: api/registerOrganisation/index.ts (hunk 2 above)
Comment (andywilkinshmcts) [reply api/registerOrganisation/index.ts:124]: I think you need to make sure that no stack trace is ever returned. What happens if the backend returns some other error other than the 2 you are handling

Comment (olusegz07) [reply api/registerOrganisation/index.ts:125]: I have updated it with a generic message.

**Review by andywilkinshmcts — APPROVED** [review]

**Review by andywilkinshmcts — DISMISSED** [review]

**Review by RiteshHMCTS — APPROVED** [review]

**Review by RiteshHMCTS — APPROVED** [review]

**Review by OgunyemiO — APPROVED** [review]

**Pull request merged**